x-pack/plugins/cases/server/services/connector_mappings/index.ts

```diff
@@ -18,16 +18,24 @@ import type {
   PostConnectorMappingsArgs,
   UpdateConnectorMappingsArgs,
 } from './types';
-import type { ConnectorMappingsPersistedAttributes } from '../../common/types/connector_mappings';
-import type { ConnectorMappings } from '../../../common/api';
+import type {
+  ConnectorMappingsPersistedAttributes,
+  ConnectorMappingsSavedObjectTransformed,
+  ConnectorMappingsTransformed,
+} from '../../common/types/connector_mappings';
+import {
+  ConnectorMappingsRt,
```

Comment:
nit: Maybe we can do `export type ConnectorMappingsTransformedRt = ConnectorMappingsRt;` in `x-pack/plugins/cases/server/common/types/connector_mappings.ts` and use it here instead of `ConnectorMappingsRt`.